nvflare/lighter/project.yml

```diff
@@ -87,6 +87,9 @@ builders:
     args:
       base_image: python:3.8
       requirements_file: docker_compose_requirements.txt
+  - path: nvflare.lighter.impl.helm_chart.HelmChartBuilder
+    args:
+      docker_image: localhost:32000/nvfl-min:0.0.1
```

Comment:
how do we get this port and image name?

Reply:
The person who is going to run helm install knows where the image is.  This setting, localhost:32000, is a typical docker registry for microk8s.

Reply:
got it, we will need some documentation updates to explain these stuff to the users.